src/boolean-contains/index.js

```diff
@@ -227,3 +245,18 @@ export function compareCoords(pair1, pair2) {
 export function getMidpoint(pair1, pair2) {
     return [(pair1[0] + pair2[0]) / 2, (pair1[1] + pair2[1]) / 2];
 }
+
+function extractPolygonBorderAsLineString(feature) {
+    const coords = getGeom(feature).coordinates;
+    const outerRing = coords[0];
+
+    // All polygons should have the same first/last point, but this is not required by GeoJSON, so add it if it is missing.
+    const firstPoint = outerRing[0];
```

Comment:
According to the [GeoJSON spec](http://geojson.win/#section-3.1.6) (see dotpoint 2) a ring must be closed so I think we could probably skip this step.

Reply:
Perhaps an earlier version of the spec...